include/openmc/constants.h

```diff
@@ -338,7 +338,8 @@ enum class RunMode {
   EIGENVALUE,
   PLOTTING,
   PARTICLE,
-  VOLUME
+  VOLUME,
+  VOLUME_NO_XS
```

Comment:
I'd prefer that we don't create an entirely new run mode for this. Instead, you can just have a check on whether cross section data is available (e.g., by checking if `settings::path_cross_sections` is empty).

Reply:
My first implementation didn't include this.
I was not convinced it was the best solution, but I wanted to test it, to see how it looked.

Reply:
thx for the feedback